tests/utils/eval_client/test_lambda_eval_client.py

```diff
@@ -0,0 +1,16 @@
+# Copyright 2022 MosaicML Composer authors
+# SPDX-License-Identifier: Apache-2.0
+import pytest
+
+from composer.utils import LambdaEvalClient
+
+
+@pytest.mark.remote
+@pytest.mark.parametrize(
+    'code, result', [['def add_1(x):\n    return x + 1', True], ['def add_1(x):\n    return y + 1', False],
+                     ['def add_1(x):\n    while True:\n        x += 1', False], ['def add_1(x): return x + 2', False]])
+def test_lambda_invoke(code, result):
```

Comment:
```suggestion
@pytest.mark.parametrize(
    'code, result', [['def add_1(x):\n    return x + 1', True], ['def add_1(x):\n    return y + 1', False],
                     ['def add_1(x):\n    while True:\n        x += 1', False], ['def add_1(x): return x + 2', False],],)
def test_lambda_invoke(code, result):
```
if you add a trailing comma, the linter will format in a way that's a lot easier to read imo